src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/ComputeTestCommon.ps1

```diff
@@ -284,9 +293,37 @@ function Get-DefaultVMConfig
 
     # VM Profile & Hardware
     $vmsize = Get-DefaultVMSize $location;
-    $vmname = Get-RandomItemName 'pstestvm';
+    $vmname = Get-RandomItemName 'crptestps';
 
     $vm = New-AzureVMConfig -VMName $vmname -VMSize $vmsize;
 
     return $vm;
-}
\ No newline at end of file
+}
+
+# Assert Output Contains
+function Assert-OutputContains
+{
+    param([string] $cmd, [string[]] $sstr)
+    
+    $st = Write-Verbose ('Running Command : ' + $cmd);
+    $output = Invoke-Expression $cmd | Out-String;
+
+    $max_output_len = 1500;
```

Comment:
Is this just an arbitrary number?  Can you add comment about why you choose this number for max_output_len?